Bridge/Exporter/AdminExporter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Bridge\Exporter;
+
+use Exporter\Exporter;
+use Sonata\AdminBundle\Admin\AdminInterface;
+
+final class AdminExporter
```

Comment:
@sonata-project/contributors : if you have a better name for this service I'm all ears. I extracted this from `CrudController::exportAction` so that it can be easily unit tested.